README.md

```diff
@@ -63,21 +63,45 @@ Reference: https://docs.travis-ci.com/user/environment-variables/#encrypting-env
 
```

Comment:
Note to self: Should update the README to include deployment instructions and the corrected version of how to add Travis encrypted variables.

Reply:
Done.